CHANGELOG.md

```diff
@@ -1,5 +1,23 @@
 # PayPal Android SDK Release Notes
 
+```kotlin 
```

Comment:
This should probably be in a migration guide rather than the CHANGELOG. We don't typically place code snippets in our release notes.